### PR TITLE
[9.0][MIG] mgmtsystem_nonconformity_project

### PR DESCRIPTION
**Pull request by thinkwelltwd**
#92

### CODE REVIEW

**Comment by max3903** [comment]
Hi @thinkwelltwd 

Let us know when you are ready to get reviewed.

**Comment by thinkwelltwd** [comment]
I'd say it's ready to review now, although test coverage declined. Is that a deal-breaker for merging?

File: mgmtsystem_nonconformity_project/hooks.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+
```
Comment (dreispt) [inline mgmtsystem_nonconformity_project/hooks.py]: Add short authorship/copyright notice (see https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py#L2-L3)

File: mgmtsystem_nonconformity_project/models/mgmtsystem_nonconformity_project.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from openerp import api, models, fields
```
Comment (dreispt) [inline mgmtsystem_nonconformity_project/models/mgmtsystem_nonconformity_project.py]: Add short authorship/copyright notice

File: mgmtsystem_nonconformity_project/models/mgmtsystem_nonconformity_project.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    def _compute_complete_name(self):
+        for action in self:
+            action.complete_name = '%s %s' % action.name_get()[0]
+
+    name = fields.Char('Claim Subject', size=128)
+    action_type = fields.Selection([
+        ('action', 'Action'),
+        ('project', 'Project'),
+    ],
```
Comment (dreispt) [inline mgmtsystem_nonconformity_project/models/mgmtsystem_nonconformity_project.py]: Isn't this underindented?

File: mgmtsystem_nonconformity_project/models/mgmtsystem_nonconformity_project.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+
+class MgmtsystemAction(models.Model):
+    _inherit = "mgmtsystem.action"
+
+    def _compute_complete_name(self):
+        for action in self:
+            action.complete_name = '%s %s' % action.name_get()[0]
+
+    name = fields.Char('Claim Subject', size=128)
```
Comment (dreispt) [inline mgmtsystem_nonconformity_project/models/mgmtsystem_nonconformity_project.py]: Remove size

File: mgmtsystem_nonconformity_project/models/mgmtsystem_nonconformity_project.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        ('project', 'Project'),
+    ],
+        string='Action Type',
+        required=True,
+        default='action',
+    )
+    project_id = fields.Many2one('project.project', 'Project')
+    complete_name = fields.Char('Complete Name',
+                                compute='_compute_complete_name',
+                                size=250, store=True)
```
Comment (dreispt) [inline mgmtsystem_nonconformity_project/models/mgmtsystem_nonconformity_project.py]: Remove size

File: mgmtsystem_nonconformity_project/models/mgmtsystem_nonconformity_project.py [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+    @api.multi
+    def name_get(self):
+        res = list()
+        for o in self:
+            r = (o.id, o.name)
+            if o.action_type == 'project' and o.project_id:
+                r = (o.id, o.project_id.name)
+            res.append(r)
+        return res
```
Comment (dreispt) [inline mgmtsystem_nonconformity_project/models/mgmtsystem_nonconformity_project.py]: I see that this is preexisting code, but I don't see the point to replace the Action name by the corresponding Project name. 
@max3903 can we just drop this?

**Review by dreispt — COMMENTED** [review]
LGTM, only a few nitpicks to improve

**Comment by dreispt** [comment]
@thinkwelltwd Can you address my comments please?

**Comment by thinkwelltwd** [comment]
Did you want the complete_name field yanked altogether in the model & views, or just set the complete_name to the value of the "name" field?

**Pull request closed without merging**